packages/compilers/test/solidityCompiler.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { expect } from 'chai';
+import {
+  getSolcExecutable,
+  getSolcJs,
```

Comment:
I see you didn't include `VyperCompiler.spec.ts`

Reply:
Ah ok, I see you just included existing tests. Let's implement the Vyper tests in a separate PR